eng/Version.Details.xml

```diff
@@ -29,21 +29,21 @@
       <Uri>https://github.com/microsoft/usvc-apiserver</Uri>
       <Sha>9582c787e7963a2acf11111e5bc5a4938e7fe3ab</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Http.Resilience" Version="9.0.0-preview.9.24524.6">
-      <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>7fe79a39d7e28f46eaa51d82a3d433bff73bd3cd</Sha>
+    <Dependency Name="Microsoft.Extensions.Http.Resilience" Version="9.0.0">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-extensions</Uri>
+      <Sha>0968e75afa659e14f00e19085475a529c29c7b59</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.TimeProvider.Testing" Version="9.0.0-preview.9.24524.6">
-      <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>7fe79a39d7e28f46eaa51d82a3d433bff73bd3cd</Sha>
+    <Dependency Name="Microsoft.Extensions.TimeProvider.Testing" Version="9.0.0">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-extensions</Uri>
+      <Sha>0968e75afa659e14f00e19085475a529c29c7b59</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Diagnostics.Testing" Version="9.0.0-preview.9.24524.6">
-      <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>7fe79a39d7e28f46eaa51d82a3d433bff73bd3cd</Sha>
+    <Dependency Name="Microsoft.Extensions.Diagnostics.Testing" Version="9.0.0">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-extensions</Uri>
+      <Sha>0968e75afa659e14f00e19085475a529c29c7b59</Sha>
     </Dependency>
     <Dependency Name="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0">
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
-      <Sha>5535e31a712343a63f5d7d796cd874e563e5ac14</Sha>
+      <Sha>9cb3b725e3ad2b57ddc9fb2dd48d2d170563a8f5</Sha>
```

Comment:
How did the Sha change, but the Version didn't?

Reply:
When you do a `darc update-dependencies --id barid` it will go and make sure you are in all the latest versions according to that barid and then update the SHAs as well. dotnet/runtime doesn't always build packages unless they are being serviced, so in those cases only the shas get updated.